demo/src/app/components/+typeahead/typeahead-section.list.ts

```diff
@@ -69,8 +69,8 @@ export const demoComponentContent: ContentSection[] = [
         title: 'With delay',
         anchor: 'delay',
         description: `
-          <p>Use to set minimal waiting time after last character typed before typeahead kicks-in.
-          In example a search begins with delay in 1 second</p>
+          <p>Use <code>typeaheadWaitMs</code> to set minimal waiting time after last character typed
+          before typeahead kicks-in. In example a typeahead appears with delay in 1 second</p>
```

Comment:
`search begins` is better for this demo IMO. a dropdown with suggestions might not appear at all